examples/multi-threading.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: UTF-8 -*-
+
+from fbchat import Client
+from fbchat.models import *
+import threading
+
+# Subclass fbchat.Client and override required methods
+class PrintMessage(Client):
+    def onMessage(self, author_id, message_object, thread_id, thread_type, **kwargs):
+        self.markAsDelivered(thread_id, message_object.uid)
+        self.markAsRead(thread_id)
+
+        print("\nIncoming message: {}".format(message_object.text))
+
+    # Creating some feedback on login since we will disable INFO logging
+    def onLoggedIn(self, email=None):
+        print("Login of {} successful.".format(email))
+
+
+# Logging in and setting logging level to WARNING to avoid some unessential output
+client = PrintMessage("<email>", "<password>", logging_level=30)
+
+
+def send():
```

Comment:
This should probably be the main thread, while the listener is spawned in a child process (Instead of spawning two child processes)

Reply:
That way you could also easily stop the listener when needed